nipype/interfaces/base.py

```diff
@@ -1258,17 +1259,17 @@ def _process(drain=0):
         stdout, stderr = proc.communicate()
         if stdout and isinstance(stdout, bytes):
             try:
-                stdout = stdout.decode()
+                stdout = stdout.decode(locale.getdefaultlocale()[1])
             except UnicodeDecodeError:
                 stdout = stdout.decode("ISO-8859-1")
         if stderr and isinstance(stderr, bytes):
             try:
-                stderr = stderr.decode()
+                stderr = stderr.decode(locale.getdefaultlocale()[1])
             except UnicodeDecodeError:
                 stderr = stderr.decode("ISO-8859-1")
 
-        result['stdout'] = str(stdout).split('\n')
-        result['stderr'] = str(stderr).split('\n')
+        result['stdout'] = text_type(stdout).split(text_type('\n'))
```

Comment:
So - I think should should be:

```
result['stdout'] = stdout.split('\n')
```

Same idea for the other uses of `text_type`.